### ISSUE TITLE
Save DBScan Model

### ISSUE THREAD

**Issue by EricCacciavillani** [annoviko/pyclustering#650]
First off thank you so much for writing this package! Been having a blast with it and the documentation is amazing! 

I found a bug though when I try to pickle.load the object back from a pickle file and it results in this error.

![image](https://user-images.githubusercontent.com/16104771/98471936-95299d00-21bd-11eb-9993-60472dbb4033.png)

**Comment by annoviko** [comment]
Hello @EricCacciavillani ,

Thank you for your reporting. Yes, I can recognize it. It because pyclustering uses C++ code to get maximum performance (I rewrite some of the algorithms using C++) and not all internal data is transferred back to python, only results. And as a result object cannot be restored. I think this is an issue for almost all algorithms which delegate calculation to C++ implementation.

I will investigate the issue and will introduce correction for all clustering algorithms.

Still, it would be useful to have a code example, I would like to know the use-case if it is possible.

**Comment by EricCacciavillani** [comment]
Sorry for responding so late was helping a co-worker. 
```Python
from eflow.utils.sys_utils import load_pickle_object
from pyclustering.cluster.dbscan import dbscan
import pickle

# Data that has had pca and scaled
scaled = auto_cluster.get_scaled_data()

# Model instance
dbscan_instance = dbscan(scaled, .07865, 8, True)
dbscan_instance.process()

# Save model to dir
file_dir = 'DBScan.pkl'
list_pickle = open(file_dir, 'wb')
pickle.dump(dbscan_instance,list_pickle)
list_pickle.close()

# Load model in
with open(file_dir, 'rb') as handle:
    load_model = pickle.load(handle)
```

**Comment by annoviko** [comment]
I have supported dumping and loading of the algorithm. The changes are on the branch `0.10.dev`, and they will be available in the next release `0.10.1`.
